beacon_node/http_api/src/lib.rs

```diff
@@ -2127,6 +2127,45 @@ pub fn serve<T: BeaconChainTypes>(
             })
         });
 
+    // GET lighthouse/seen_validators?ids,epochs
+    let get_lighthouse_seen_validators = warp::path("lighthouse")
+        .and(warp::path("seen_validators"))
+        .and(warp::path::end())
+        .and(warp::query::<api_types::SeenValidatorQuery>())
+        .and(chain_filter.clone())
+        .and_then(
+            |query: api_types::SeenValidatorQuery, chain: Arc<BeaconChain<T>>| {
+                blocking_json_task(move || {
+                    let indices: Vec<usize> = query
+                        .ids
+                        .0
+                        .iter()
+                        .map(|id| match id {
+                            ValidatorId::PublicKey(pubkey) => chain
+                                .validator_index(pubkey)
+                                .map_err(warp_utils::reject::beacon_chain_error)?
+                                .ok_or_else(|| {
+                                    warp_utils::reject::custom_not_found(format!(
+                                        "Could not find index for validator public key: {}",
```

Comment:
```suggestion
                                        "could not find index for validator public key: {}",
```

Potentially more consistent with other server errors.